src/Communication/readme.md

```diff
@@ -0,0 +1,74 @@
+<!-- region Generated -->
+# Az.Communication
+This directory contains the PowerShell module for the Communication service.
+
+---
+## Status
+[![Az.Communication](https://img.shields.io/powershellgallery/v/Az.Communication.svg?style=flat-square&label=Az.Communication "Az.Communication")](https://www.powershellgallery.com/packages/Az.Communication/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.8.1 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.Communication`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
+  - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/communication/resource-manager/Microsoft.Communication/preview/2020-08-20-preview/CommunicationService.json
```

Comment:
Please stick to a commit rather master branch

Reply:
Updated to point to a specific commit